src/lib/packager/clients/pkg_finish.rb

```diff
@@ -145,14 +145,14 @@ def backup_all_target_sources
     #
     # @return [Array<Y2Packager::Repository>] List of disabled repositories
     def disable_local_repos
-      candidates_repos, other_repos = *::Y2Packager::Repository.enabled.partition(&:local?)
-      products = other_repos.map(&:products).flatten.uniq
+      candidates_repos, remote_repos = *::Y2Packager::Repository.enabled.partition(&:local?)
```

Comment:
NP: I think you could rename `candidates_repos` to `local_repos` so we have `local_repos` and `remote_repos` pair.